flake.nix

```
@@ -0,0 +1,52 @@
+{
+  description = "Rustlings devShell.";
```

Might be better to use the description in this repo.

```suggestion
  description = "Small exercises to get you used to reading and writing Rust code";
```

---

:+1: